addon/components/o-s-s/radio-button.hbs

```diff
@@ -0,0 +1,3 @@
+<button class="upf-radio-btn {{if @selected 'upf-radio-btn--selected'}} {{if @disabled 'upf-radio-btn--disabled'}}"
```

Comment:
pretty sure we have `upf-radio-btn` being used in the software already so just be careful with that one 😅